lms/djangoapps/courseware/views/index.py

```diff
@@ -183,7 +183,7 @@ def _verify_position(self):
         """
         if self.position is not None:
             try:
-                int(self.position)
+                self.position = max(int(self.position), 1) # test change
```

Comment:
If this comment is supposed to indicate that this change needs to be tested, you can add a TODO comment to make that clear.